kedro/io/partitioned_dataset.py

```diff
@@ -40,68 +40,63 @@ class PartitionedDataSet(AbstractDataSet):
     `lazy saving <https://kedro.readthedocs.io/en/stable/data/\
     kedro_io.html#partitioned-dataset-lazy-saving>`_.
 
-    Example adding a catalog entry with
+    Example using the
     `YAML API <https://kedro.readthedocs.io/en/stable/data/\
-        data_catalog.html#use-the-data-catalog-with-the-yaml-api>`_:
+    data_catalog.html#use-the-data-catalog-with-the-yaml-api>`_:
 
     .. code-block:: yaml
 
-        >>> cv_results_partitioned: # example to save results to multiple partitions
```

Comment:
This makes it seem like you need to define a separate dataset for saving and loading; I've consolidated into one example.